ImplantFramework/lib/source/transports/linux_tcp.hpp

```diff
@@ -74,17 +76,20 @@ class LinuxTcpTransport : public hivemind_lib::Transport {
     int socket = this -> ConnectTo();
     if (socket < 0) {
       DEBUG("ERROR WITH SOCKET", LEVEL_ERROR);
+      goto cleanup;
```

Comment:
Not sure if calling close on a nonexisting socket handle will make the system very happy. We'll have to look out for this.